web/src/utils/retryImport.ts

```diff
@@ -0,0 +1,30 @@
+import { ComponentType } from 'react'
```

Comment:
Did you have a look at [this proposal](https://gist.github.com/raphael-leger/4d703dea6c845788ff9eb36142374bdb#file-lazywithretry-js) from [this article](https://raphael-leger.medium.com/react-webpack-chunkloaderror-loading-chunk-x-failed-ac385bd110e0)? The approach is a little bit different, he also says one should only try twice and throw an error if that does not help. 

He reloads the whole page why you are just retrying the import. I am no expert here, could you explain why you chose this solution? Reproducing/testing this is sadly very difficult :/

Reply:
Well because this particular chunk failed to load during the lazy import so for me it makes sense to retry importing/loading this chunk again if it fails.
Well there are very recent articles about that case also here f.e.
https://medium.com/@botfather/react-loading-chunk-failed-error-88d0bb75b406
You can also find this solving approach in further blogs and git issues since this feature was released with React 16.6
And yes testing is difficult. I even could never reproduce this error.

And we could reduce the retry attempts. I think three can be fine, too.